### PR TITLE
fix simple_project example

### PR DESCRIPTION
**Pull request by ivan-savciuc**
## About this change—what it does

fixes sample project version

Resolves: #799

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ nav_order: 1
 
 ## [X.X.X] - not yet released
 - Fix auto generated documentation by bumping tfplugindocs to latest version
-- Fix Typos in docs
+- Fix Typos in docs and examples
```
Comment (Serpentiel) [inline CHANGELOG.md:10]: can we lowercase the word _typos_?

**Review by Serpentiel — DISMISSED** [review]
lgtm, but see comments

**Review by mhoffm-aiven — DISMISSED** [review]
lgtm

**Comment by ivan-savciuc** [comment]
Updated `Typos` -> `typos`

**Review by Serpentiel — APPROVED** [review]
lgtm

**Pull request merged**